chain-core/src/init/network.rs

```diff
@@ -114,3 +115,10 @@ mod test {
         assert_eq!(1, get_bip44_coin_type_from_network(Network::Devnet));
     }
 }
+
+
+#[no_mangle]
+pub extern "C" fn export_dev_network(net: Network) -> Network  
+{
+    return net;
+}
```

Comment:
oh, this one is because of cbindgen? I guess you can just export a useful function instead -- e.g. `get_bip44_coin_type_from_network` (just add `#[no_mangle]` and `extern` to it)

Reply:
yes, this is for cbindgen exporting.
i'll add useful function